src/target_info.rs

```diff
@@ -5,10 +5,21 @@ pub const RISCV_ARCH_MAPPING: &[(&str, &str)] = &[
     ("riscv32gc", "riscv32"),
     ("riscv32i", "riscv32"),
     ("riscv32im", "riscv32"),
-    ("riscv32ima", "riscv32"),
     ("riscv32imac", "riscv32"),
     ("riscv32imafc", "riscv32"),
     ("riscv32imc", "riscv32"),
     ("riscv64gc", "riscv64"),
     ("riscv64imac", "riscv64"),
 ];
+pub const WINDOWS_TRIPLE_MAPPING: &[(&str, &str)] = &[
```

Comment:
No idea if I should skip `uwp` and `win7` here.

Reply:
I think it's fine